dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/common/jaxb/JAXBContextCache.java

```
@@ -0,0 +1,504 @@
+                        = new CacheMap<>();
+                }
+                cachedContextAndSchemasInternalMap.put((defaultNs != null) ? defaultNs : "",
+                    cachedContextAndSchemasInternal);
+                JAXBCONTEXT_CACHE.put(classes, cachedContextAndSchemasInternalMap);
+            }
+        }
+
+        return new CachedContextAndSchemas(context, classes, cachedContextAndSchemasInternal);
+    }
+
+    private static boolean checkObjectFactoryNamespaces(Class<?> clz) {
+        for (Method meth : clz.getMethods()) {
+            XmlElementDecl decl = meth.getAnnotation(XmlElementDecl.class);
+            if (decl != null
+                && XmlElementDecl.GLOBAL.class.equals(decl.scope())
+                && StringUtils.isEmpty(decl.namespace())) {
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+
+    private static JAXBContext createContext(final Set<Class<?>> classes,
+                                      final Map<String, Object> map,
+                                      Collection<Object> typeRefs)
+        throws JAXBException {
+        boolean isLoggableInfo = LOG.isLoggable(Level.INFO);   // Liberty Change
+        JAXBContext ctx;
+        if (typeRefs != null && !typeRefs.isEmpty()) {
+            Class<?> fact = null;
+            String pfx = "com.sun.xml.bind.";
+            try {
+                fact = ClassLoaderUtils.loadClass("com.sun.xml.bind.v2.ContextFactory",
+                                                  JAXBContextCache.class);
+            } catch (Throwable t) {
+                try {
+                    fact = ClassLoaderUtils.loadClass("com.sun.xml.internal.bind.v2.ContextFactory",
+                                                      JAXBContextCache.class);
+                    pfx = "com.sun.xml.internal.bind.";
+                } catch (Throwable t2) {
+                    //ignore
+	            // Liberty Change begin
+	            if (isLoggableInfo) {
```

Since this exception is ignored, this should be logged only in the trace.log. Check and log needs to change to `fine`